Assets/HoloToolkit/Common/Shaders/Standard.shader

```diff
@@ -118,6 +118,8 @@ Shader "MixedRealityToolkit/Standard"
             #pragma shader_feature _BORDER_LIGHT_OPAQUE
             #pragma shader_feature _INNER_GLOW
             #pragma shader_feature _ENVIRONMENT_COLORING
+            
+            #define IF(a, b, c) lerp(b, c, step((fixed) (a), 0));
```

Comment:
Can we change 0 to 0.0 to avoid conversion on some platforms? For example: #define IF(a, b, c) lerp(b, c, step((fixed) (a), 0.0));

Reply:
I did it in an update now.